src/cli.rs

```diff
@@ -96,7 +100,16 @@ pub struct Cli {
 
 impl Cli {
     pub fn from_args() -> Self {
-        Cli::from_iter(std::env::args())
+        let mut args = Cli::from_iter(std::env::args());
+        if args.body {
```

Comment:
Ah, interesting, this was wat I initially thought of doing and thought it would be cleaner here.
sure, I can move it to `Print::new()` I'll fix it and push later today!

Thanks!